mathlibtools/lib.py

```diff
@@ -842,12 +843,20 @@ def reduce_imports_sed(self, file: str) -> Iterable[str]:
 
     def make_all(self) -> None:
         """Creates all.lean importing everything from the project"""
+
+        # helper function to wrap file paths in double quotes if they
+        # contain dashes, so that lean accepts them as imports
+        def make_safe(s):
+            if re.match(LEAN_UNESCAPED_IDENTIFIER_RE, s):
```

Comment:
Note that `match` matches at the start of the string, making the `^` in the regex redundant. If you use `fullmatch` instead, then the `$` is redundant too.

Reply:
cool thanks